pandas/core/tools/numeric.py

```diff
@@ -110,6 +110,21 @@ def to_numeric(arg, errors="raise", downcast=None):
     2    2.0
     3   -3.0
     dtype: float64
+
+    Downcasting of ``ExtensionDtype`` is supported:
```

Comment:
```suggestion
    Downcasting of nullable integer and floating dtypes is supported:
```

(general extension dtypes are not supported)